modules/openapi-generator/src/main/resources/cpp-qt5-client/HttpRequest.cpp.mustache

```diff
@@ -313,11 +305,8 @@ void {{prefix}}HttpRequestWorker::execute({{prefix}}HttpRequestInput *input) {
         buffer->setParent(reply);
 #endif
     }
-    if(timeout > 0){
-        timer->setSingleShot(true);
-        timer->setInterval(timeout);
-        connect(timer, &QTimer::timeout, this, [=](){ on_manager_timeout(reply); });
-        timer->start();
+    if(_timeOut > 0){
+        QTimer::singleShot(_timeOut, [&](){ on_manager_timeout(reply); });
```

Comment:
What happens if the timer elapsed at the same time response is received? Would there be a crash because the object is still referenced in the Qt event Loop? Won't this happen since the QNetworkReply is async?

Reply:
In #4430, I replaced [&] by [=]